docker/logstash/etc/logstash/conf.d/logstash.conf.example

```diff
@@ -1,45 +0,0 @@
-input {
```

Comment:
why did you remove example?

Reply:
Because it isn't used any more. This config is generated by corresponding template.

Reply:
but this one is an example file, not a config itself

Reply:
Ok, reverted file back